docs/book/src/architecture/controllers/cluster.md

```diff
@@ -1 +1,45 @@
-# Cluster
+# Cluster Controller
+
+The cluster controller is responsible for keeping the cluster object in sync with the infrastructure cluster object.
+
+Its main responsibilities are:
+
+* Setting an OwnerReference on the infrastructure object.
+* Deleting all related objects in the correct order.
+* Keeping the Cluster object up to date with external objects.
+* Creating a kubeconfig secret for child clusters.
```

Comment:
```suggestion
* Creating a kubeconfig secret for [target clusters](link to definition).
```

Reply:
I thought the term of art was "workload cluster," did that change?

Reply:
workload cluster is right

Reply:
I've seen workload clusters and target clusters used interchangeably. I'll go with the suggestion